<launch>

  <arg name="model_path" default="/home/docker_track/deep_annotation_inference.h5" />
  
  <arg name="points" default="/depth_registered/points" />
  <arg name="objects" default="/stereo/left/track_ids" />
  <arg name="markers" default="/pc_to_bbox/visualisation" />
  <arg name="clear_pc" default="/pc_to_bbox/clear_pc" />

  <remap from="point_cloud" to="$(arg points)" />
  <remap from="objects" to="$(arg objects)" />
  <remap from="visualisation" to="$(arg markers)" />
  <remap from="clear_pc" to="$(arg clear_pc)" />

  <node pkg="pc_to_bbox" type="pc_to_bbox.py" name="pc_to_bbox" output="screen" >
    <param name="model_path" value="$(arg model_path)" />
  </node>

</launch>
